disease-classification-training/unittests.py

```diff
@@ -38,3 +42,29 @@ def train_params(self):
     def __str__(self):
         return "TrainAccError: Model train accuracy/loss is not within acceptable range"
 
+class TrainSpecError(TestDiseaseClassification):
```

Comment:
Not sure what is going on with they classes? Let's discuss on Monday. 

I was expecting a unittest frame work style classes?

Reply:
These classes are similar to the tests I've written for connectors